src/pages/articles/bootstrap/bootstrap-buttons/index.md

```diff
@@ -2,14 +2,36 @@
 title: Bootstrap Buttons
 ---
 ## Bootstrap Buttons
+The Bootstrap framework provides you with various styling options for buttons. These styles help you provide a visual representation to the user of what the button may do.
 
-This is a stub. [Help our community expand it](https://github.com/freecodecamp/guides/tree/master/src/pages/articles/bootstrap/bootstrap-buttons/index.md).
+#### How To Use:
+To use bootstrap buttons you follow the same steps that you would to create a button in HTML except you also apply the applicable CSS class to the button. A code example has been provided below.
 
-[This quick style guide will help ensure your pull request gets accepted](https://github.com/freecodecamp/guides/blob/master/README.md).
+**Code Example:**
 
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+<button type="button" class="btn btn-primary">Primary</button>
+`<button type="button" class="btn btn-primary">Primary</button>`
 
-#### More Information:
-<!-- Please add any articles you think might be helpful to read before writing the article -->
+#### Bootstrap Button Class List:
+This is a list of the CSS classes that bootstrap provides for buttons.
+
+`.btn` <button type="button" class="btn">Basic</button> This is bootstrap's basic button.
```

Comment:
Can we have the button element on a new line after `This is bootstrap's ${buttonType} button`

And the same for each example :+1: